code/drasil-docLang/lib/Drasil/Sections/ReferenceMaterial.hs

```diff
@@ -10,3 +10,7 @@ import Language.Drasil
 -- | Default Reference Material section introduction.
 intro :: Contents
 intro = mkParagraph $ S "This section records information for easy reference."
+
+-- | Helper to create default sentence for empty sections using NamedIdea
+emptySectSentence :: NamedIdea n => n -> Contents
+emptySectSentence var = foldlSP [S "There are no" +:+ plural var]
```

Comment:
If this took a list of `NamedIdea`s, we would be able to use it when there are multiple (e.g., "There are no <x>s or <y>s.") using `foldlList Comma Options`.

Reply:
Thats a good idea! I will change that right now!